shenyu-web/src/main/java/org/apache/shenyu/web/handler/ShenyuWebHandler.java

```diff
@@ -105,7 +104,7 @@ public Mono<Void> handle(@NonNull final ServerWebExchange exchange) {
      *
      * @param extPlugins the ext plugins
      */
-    public void putExtPlugins(final List<ShenyuPlugin> extPlugins) {
+    public synchronized void putExtPlugins(final List<ShenyuPlugin> extPlugins) {
```

Comment:
used volatile， maybe not used synchronized？

Reply:
This method can be called without synchronized because it will not be called concurrently, I remove this